front-end/src/App.css

```diff
@@ -36,3 +36,22 @@
     transform: rotate(360deg);
   }
 }
+
+.parent {
+  height: 100px;
+  width: 300px;
+  position: relative;
+}
+
+.child {
+  position: absolute;
+  top: 50%;
+  transform: translate(280%,-10%);
```

Comment:
This looks funky, what are we trying to achieve here?
If you trying to center an element, there should be a better way to do it.

Reply:
I agree. I've tried a lot of methods but nothing seems to center the title accurately. I think the problem is that the cloudspace Header tag does'nt respond well to such methods. However I did this but there is one drawback, the component does'nt center according to the screen size.
```
.parent {
  height: 120px;
  width: 1500px;
  top: 15%;
  position: relative;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-left: auto;
  margin-right: auto;
}
```